hilti/src/rt/util.cc

```diff
@@ -50,13 +51,19 @@ void hilti::rt::abort_with_backtrace() {
 
 void hilti::rt::cannot_be_reached() { hilti::rt::internalError("code is executing that should not be reachable"); }
 
-hilti::rt::MemoryStatistics hilti::rt::memory_statistics() {
-    MemoryStatistics stats;
+hilti::rt::ResourceUsage hilti::rt::resource_usage() {
+    ResourceUsage stats;
 
     struct rusage r;
     getrusage(RUSAGE_SELF, &r);
     auto fibers = detail::Fiber::statistics();
 
+    const auto to_double = [](const auto& x) {
+        return static_cast<double>(x.tv_sec) + static_cast<double>(x.tv_usec) / 1e6;
+    };
```

Comment:
Maybe we could be clearer on types and units here, e.g.,
```suggestion
        const auto to_seconds = [](const timeval& t) {
        return static_cast<double>(t.tv_sec) + static_cast<double>(t.tv_usec) / 1e6;
    };
```